### PR TITLE
update dsp version 1.7.1

### PR DESCRIPTION
**Pull request by charmingduchess**

### CODE REVIEW

**Review by nonword — APPROVED** [review]

File: test/discovery-store-model-test.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const sinon = require('sinon')
 const AWS = require('aws-sdk')
 
-const HoldingsUpdater = require('pcdm-store-updater/lib/holdings-updater')
+// const HoldingsUpdater = require('pcdm-store-updater/lib/holdings-updater')
```
Comment (nonword) [inline test/discovery-store-model-test.js:4]: Should prob remove this

File: test/discovery-store-model-test.js [hunk 2]
```diff
@@ -346,16 +346,24 @@ describe('discovery-store-model', () => {
...
-
       expect(groupedStatements).to.be.a('array')
       expect(groupedStatements[0]).to.be.a('object')
       expectStatement(groupedStatements[0]._statements, {
         predicate: 'dcterms:title',
         object_literal: 'AAHGS news : the bi-monthly newsletter of the Afro-American Historical and Genealogical Society, Inc.'
       })
-
+      const realItems = groupedStatements[0]._items.filter(i => !i.id.includes('i-h'))
+      const checkInCardItems = groupedStatements[0]._items.filter(i => i.id.includes('i-h'))
```
Comment (nonword) [inline test/discovery-store-model-test.js:356]: This is fine. It relies on the convention for generating checkin card ids, which will be pretty static, but conceivably could change. So you could use the `rdfs:type` check from below to separate groups, e.g.:
```
const isCheckinCard = (item) => item._statements.some((s) => {
  return s.predicate === 'rdfs:type' &&
    s.object_id === 'nypl:CheckinCardItem'
})
const realItems = groupedStatmeents[0]._items.filter((g) => !isCheckinCard(g))
const checkinCardItems = groupedStatements[0]._items.filter(isCheckinCard)
```

Comment (charmingduchess) [reply test/discovery-store-model-test.js]: added

File: test/discovery-store-model-test.js (hunk 1 above)
Comment (charmingduchess) [reply test/discovery-store-model-test.js]: removed

**Review by nonword — APPROVED** [review]

**Pull request merged**